report/chapters/foundation.typ

```diff
@@ -29,15 +73,22 @@ The absence of these features and the synchronizing problems with SAP have led t
     table.header([* Missing feature *],[* Desciption *]), 
 
       [Holiday balance],[The program has to display the individual employees current available absence days. Prefereably there should be a function to loan absence days from the future which then are regained throughout the year.  ],
-      [Lookup specific team],[The current system is missing a way to quickly lookup a specific team in any department, there should be a function to search for the specific team and its members.],
-      [Create teams across departments],[Too fully utilize the teams the company should be able to create teams with members across different departments.],
+      [Lookup specific team/employee],[The current system is missing a way to quickly lookup a specific team in any department, there should be a function to search for the specific team and its members.],
+      [Create/edit teams across departments],[Too fully utilize the teams the company should be able to create teams with members across different departments. Additionally a team should be editable, to prevent the occurence of dead teams.],
       [Viewable Holidays],[When looking up other teams, the user should be able to see what days the members of said team are absent.],
-      [Toggleable UI],[The user should be able to switch between different calendar types, e.g. weekly, monthly, yearly.],
+      [Intuitive design],[The design should create a comprehensible overview for the user when viewing absence periods for both individuals and teams. This may include color coordination or additional notes.],
+      [Toggleable UI],[The user should be able to switch between different calendar types, e.g. weekly, monthly, yearly. Furthermore, the toggleability should also include options for calendar orientations, e.g. horisontal and vertical.],
```

Comment:
```suggestion
      [Toggleable UI],[The user should be able to switch between different calendar types, e.g. weekly, monthly, yearly. Furthermore, it should be possible to toggle between calendar orientations, such as horizontal and vertical.],
```

Corrected from British english to american english as per our agreement to stick to american english. Also suggestion to rewrite sentence for clearer understanding